README.md

```diff
@@ -217,16 +217,17 @@ If you want to contribute to this list (please do), send me a pull request.
 * [laravel-graphql](https://github.com/Folkloreatelier/laravel-graphql) - Facebook GraphQL for Laravel 5.
 * [laravel-graphql-relay](https://github.com/nuwave/laravel-graphql-relay) - A Laravel library to help construct a server supporting react-relay.
 * [graphql-mapper](https://github.com/4rthem/graphql-mapper) - This library allows to build a GraphQL schema based on your model.
-* [graphql-bundle](https://github.com/suribit/GraphQLBundle) - GraphQL Bundle for Symfony 2.
-* [overblog/graphql-bundle](https://github.com/overblog/GraphQLBundle) - This bundle provides tools to build a complete GraphQL server in your Symfony App. Supports react-relay.
-* [GraphQL](https://github.com/Youshido/GraphQL) – Well documented PHP implementation with no dependencies.
-* [GraphQL Symfony Bundle](https://github.com/Youshido/GraphQLBundle) – GraphQL Bundle for the Symfony 3 (supports 2.6+).
+* [Symfony 2 Bundle by hoticeking](https://github.com/hoticeking/GraphQLBundle ) - GraphQL Bundle for Symfony 2.
+* [Symfony Bundle by Overblog](https://github.com/overblog/GraphQLBundle) - This bundle provides tools to build a complete GraphQL server in your Symfony App. Supports react-relay.
+* [GraphiQL Symfony Bundle by Overblog](https://github.com/overblog/GraphQLBundle) - This bundle provides GraphiQL tool into symfony, can be compatible with any GraphQL server implementation
```

Comment:
Should be https://github.com/overblog/GraphiQLBundle

Reply:
Fixed :+1: